.jshintignore

```diff
@@ -46,6 +46,7 @@ apps/music/js/metadata_scripts.js
 apps/keyboard/js/imes/jszhuyin/tools/cook.js
 apps/keyboard/build/keyboard-config.js
 apps/settings/js/vendor/alameda.js
+apps/settings/js/jszip.js
```

Comment:
After a second though I think it would make more sense to place `jszip.js` under `apps/settings/js/vendor/`. Could you update the path? Thanks.